core/src/test/scala/integration/kafka/api/PlaintextAdminIntegrationTest.scala

```diff
@@ -2152,6 +2152,34 @@ class PlaintextAdminIntegrationTest extends BaseAdminIntegrationTest {
     }, "Expected to see the broker properties we just removed to be deleted", pause=25)
   }
 
+  // The remote storage config validation on controller level only works in KRaft
+  @ParameterizedTest
+  @ValueSource(strings = Array("kraft"))
+  def testDisablingRemoteStorageIncrementalAlterConfigs(quorum: String): Unit = {
```

Comment:
can we move the integration tests to RemoteTopicCrudTest?

Reply:
Good suggestion. Updated. Thanks.